README.md

```diff
@@ -158,10 +160,17 @@ Generate manifests e.g. CRD, RBAC etc.
 
 ## Community
 
-If you have any questions or want to contribute, you are welcome to join our
-[slack channel](https://join.slack.com/t/kruise-workspace/shared_invite/enQtNjU5NzQ0ODcyNjYzLWMzZDI5NTM3ZjM1MGY2Mjg1NzU4ZjBjMDJmNjZmZTEwYTZkMzk4ZTAzNmY5NTczODhkZDU2NzVhM2I2MzNmODc).
+If you have any questions or want to contribute, you are welcome to communicate most things via GitHub issues or pull requests.
+
+Other active communication channels:
+
+- Slack: [channel address](https://join.slack.com/t/kruise-workspace/shared_invite/enQtNjU5NzQ0ODcyNjYzLWMzZDI5NTM3ZjM1MGY2Mjg1NzU4ZjBjMDJmNjZmZTEwYTZkMzk4ZTAzNmY5NTczODhkZDU2NzVhM2I2MzNmODc)
+- Mailing List: todo
+- Dingtalk Group(钉钉讨论群)
 
-Mailing List: todo
+<div align="center">
```

Comment:
change this to markdown instead of using html?

Reply:
I wish to, while actually I found no way to support that in the github markdown.